stdlib/public/core/Misc.swift

```diff
@@ -169,6 +169,65 @@ func _rethrowsViaClosure(_ fn: () throws -> ()) rethrows {
   try fn()
 }
 
+/// A type whose values can be implicitly or explicitly copied.
+///
+/// Conforming to this protocol indicates that a type's value can be copied;
+/// this protocol doesn’t have any required methods or properties.
+/// You don't generally need to write an explicit conformance to `Copyable`.
+/// The following places implicitly include `Copyable` conformance:
+///
+/// * Structure declarations,
+///   unless it has a noncopyable stored property
+/// * Enumeration declarations,
+///   unless it has a case whose associated value isn't copyable
+/// * Class declarations
+/// * Actor declarations
+/// * Protocol declarations
+/// * Associated type declarations
+/// * The `Self` type in a protocol extension
```

Comment:
Did we consider whether to call out the generic parameters in an extension of a generic type? Something like:

```suggestion
/// * The `Self` type in a protocol extension
/// * The generic parameters of a nominal type in an extension
```

Reply:
s/nominal type/extended type/

Reply:
Changed in 6fa34bdabf7